napari/layers/image/image.py

```diff
@@ -337,6 +337,26 @@ def rendering(self, rendering):
         self._rendering = rendering
         self.events.rendering()
 
+    def _get_state(self):
+        """Get dictionary of layer state.
+
+        Returns
+        -------
+        state : dict
+            Dictionary of layer state.
+        """
+        state = self._get_base_state()
+        state['rgb'] = self.rgb
+        state['is_pyramid'] = self.is_pyramid
+        state['colormap'] = self.colormap[0]
+        state['contrast_limits'] = self.contrast_limits
```

Comment:
maybe add `contrast_limits_range` now that it's not private?

Reply:
The rule I am following for adding here or not is if it is present as a keyword arg to the layer. If we add here then we should add to the function signature of `Image` and `Surface`. I'm not sure if we want to do that or not, I think we did at one point, but that can happen later